tests/gtests/test_SW_Flow_Lib_PET.cc

```diff
@@ -589,7 +589,7 @@ namespace
   //   Comparison against examples by Duffie & Beckman 2013 are expected to
   //   deviate in value, but show similar patterns, because equations for
   //   (i) sun-earth distance equation and (ii) solar declination differ
-  TEST(SW2SolarRadiationTest, extraterrestrial)
+  TEST_F(AllTest, extraterrestrial)
```

Comment:
Please change `TEST_F(AllTest, extraterrestrial)` to something like `TEST_F(AllTest, SolarRadiationExtraterrestrial)`